pennylane_lightning/core/lightning_base.py

```diff
@@ -499,9 +500,25 @@ def loop(i, y):
             [1.0, 0.8775825618903728, 0.5403023058681395]
 
         """
-        # has jax dependency, so can't import up top
+        # jax is still an optional dependency for pennylane, but mandatory for program capture
+        # jax imports cannot be placed in the standard import path
+        import jax
+        from pennylane.capture.primitives import AbstractMeasurement
+
         from .lightning_interpreter import LightningInterpreter
 
+        if jax.config.jax_enable_x64:  # pylint: disable=no-member
+            dtype_map = {
```

Comment:
Could you add a unit test for this branch?

Reply:
tested now.